README.md

```diff
@@ -59,6 +59,16 @@ The library can be loaded either as a standalone script, or through an [AMD][amd
 </script>
 ```
 
+Through [Unpkg CDN](unpkg.com)
+The library can be loaded either as a standalone script, or through [Unpkg CDN](unpkg.com)
+
+```html
+<script type="text/javascript" src="https://unpkg.com/validator.js@latest/dist/validator.min.js"></script>
```

Comment:
I don't think that the package is right as you can see [here](https://unpkg.com/browse/validator.js@2.0.4/package.json) :)

Reply:
good catch. I think for now we can just drop plugs to the CDNs that we don't manage. The only autheritative sources for our "executables" is NPM ang Github.